lib/src/main/java/org/team1540/rooster/testers/motor/SimpleControllersTester.java

```diff
@@ -148,11 +148,13 @@ public void initSendable(SendableBuilder builder) {
 
   /**
    * Add all the sendables to the {@link SmartDashboard}. Includes chooser for selecting the
-   * active {@link IMotorController} and information about the tester.
+   * active {@link IMotorController} and information about the tester. Follows a builder pattern.
+   * @return Follows a builder pattern.
    */
-  public void addAllSendables() {
+  public SimpleControllersTester addAllSendables() {
```

Comment:
Should add this (this isn't a suggestion because it also needs an import statement):
```java
@Contract(" -> this")
```